content/blog/2021/containerqueries.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Getting Started with Container Queries
+sub: Now is the time to begin experimenting with a long requested layout tool. 
+author: davidh
+date: 2021-04-01
+image:
+  src: projects/w3c.jpg
+  alt: W3C logo
+tags:
+  - CSS
+  - CSSWG
+  - Article
+  - Container Queries
+summary: |
+  An early prototype version of **container queries** is now available in [Chrome Canary](https://www.google.com/chrome/canary). What are container queries exactly and how will they change the way we build layouts?
+---
+
+{% import 'embed.macros.njk' as embed %}
+
+ The `@container` query, that elusive feature developers have been requesting and proposing for years, has finally made its debut in browsers. Well, sort of. As of the latest version of [Chrome Canary](https://www.google.com/chrome/canary), the most recent `@container` query proprosal, developed by OddBird's own [Miriam Suzanne](https://www.oddbird.net/authors/miriam/), is available for use behind an experimental flag. What _are_ container queries exactly? And how do they work?
```

Comment:
Might be good to credit David Baron for the [original draft](https://github.com/dbaron/container-queries-implementability) here?

Reply:
Made a more direct reference to his work.